lisp/ess-custom.el

```diff
@@ -2108,27 +2113,26 @@ for help files.  The default value is nil for other systems."
 
 ;;;;; names for communication using MS-Windows 9x/NT ddeclient mechanism
 
-(defcustom inferior-ess-ddeclient nil
+(defcustom inferior-ess-ddeclient "Initial"
```

Comment:
I think we can drop support for DDE. I doubt anyone uses that anymore. No-one tests is anyways. Let's disucss this on ess-devel.


Why "Initial". Looks like a wart.